### DISCUSSION TITLE
Can it be used for React Native?

### DISCUSSION THREAD

**Discussion by jck-y** [SahilAggarwal2004/react-text-to-speech#6] [General]
I'm currently creating a tts feature project for an Android app, is it compatible with React Native?

**Comment by SahilAggarwal2004** [comment]
This library uses the `WebSpeechAPI` which is available only in browsers.
So to answer your question, this library is not compatible with React Native.
Although, there are several other libraries available on npm that support tts functionality for React Native apps. I suggest you to try those and find the best one according to your use case.
